github.com/kanban/projects.user.js

```diff
@@ -160,4 +245,4 @@ $(function() {
     addPRLinks(card);
     addMenuOptions(card);
   });
-});
+});
```

Comment:
So you are the guy that commits file without empty lines at end of file...
An I'm the guy that commits the reverse operation 😆